src/components/variable.jsx

```diff
@@ -48,67 +48,142 @@ const Variable = React.createClass({
   },
   renderVariableMetadata(variable) {
     const entityMessage = {
-      famille: "une famille",
-      foyer_fiscal: "un foyer fiscal",
-      individu: "un individu",
-      menage: "un ménage",
+      famille: "famille",
+      foyer_fiscal: "foyer fiscal",
+      individu: "individu",
+      menage: "ménage",
     }
     const definitionPeriodMessage = {
-      YEAR: "de l'année",
-      MONTH: "du mois",
-      ETERNITY: "de l'éternité (cette variable ne peut pas évoluer avec le temps)",
+      YEAR: <FormattedMessage id = "aYear"/>,
+      MONTH: <FormattedMessage id = "aMonth"/>,
+      ETERNITY: <FormattedMessage id = "forEternity"/>
     }
+    const valueTypeMessage = {
+      Int: <FormattedMessage id = "anInteger"/>,
+      Float: <FormattedMessage id = "aFloat"/>,
+      Date: <FormattedMessage id = "aDate"/>,
+    }
+    function formatDefaultValue(variable) {
+    if (variable.valueType == "Date") {
+      return <FormattedDate value = { variable.defaultValue } year = "numeric" month = "2-digit" day = "2-digit" />
+    } else if (variable.valueType == "Float" && String(variable.defaultValue).indexOf(".") == -1) {
```

Comment:
Sorry for this late additional remark, but you can make it much simpler:

```JS
} else if (variable.valueType == "Float") {
        return variable.defaultValue.toFixed(1);
    }
```

I'll change it before merging if you don't have time to do the change yourself :).